news/2024/2024-07-19-osu-lazer-updates-july-18-2024.md

```diff
@@ -0,0 +1,64 @@
+- **You can now jump to specific timestamps by pasting them in the bottom bar in the editor.** This is a very small UI change that will mean a lot to mappers, especially when it comes to modding.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/editor-timestamp.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Beatmap contents can now be edited externally.** Due to the way osu!(lazer) handles beatmaps, you were not able to just open the songs folder and edit whatever you wanted, however after today, that is possible. The sounds of cheering from mapping enthusiasts can be heard around the world.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/edit-externally.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Exported maps made in osu!(lazer) should no longer be broken if they are imported to osu!(stable).** This is yet another step that's making mapping in osu!(lazer) closer to reality.
+- **Hexagonal and circular options have been added to the snap grid.** Expanding on the snap grid editing change made last month, more options have been added which will make the mapping journey easier for both veteran and newbie mappers.
+- **Various improvements have been made to the editor UI.** Long story short, timelines have been slightly reduced, colours for timing points and kiai's have been changed, and tooltips on hover have been added to make it clear to newer players what each marker and highlighted section in the timeline mean.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/editor-tooltip.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Other changes
+
+- **The ruleset selector has new samples, and some UI samples got refreshed.** See if you can spot the UI samples that got changed!
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/sound-comparison.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **New "No Release" mod has been added to osu!mania.** This is effectively relax for osu!mania, completely removing the accuracy judgment from slider ends in gameplay.
+- **The colour of the mod multiplier has been changed to match the mod icon background.** If a mod is increasing difficulty, the mod multiplier will now be red, and if the mod is decreasing difficulty, the mod multiplier will be green.
```

Comment:
this seems very minor?

Reply:
i felt like the change was worth covering since it strays away from what ppl are used to but yea it is very minor